rpcclient/evo_test.go

```diff
@@ -540,3 +610,30 @@ func TestProTxRevoke(t *testing.T) {
 		t.Fatal("returned key was not 64 characters: ", result)
 	}
 }
+
+type roundTripperFunc func(*http.Request) (*http.Response, error)
+
+// RoundTrip ...
+func (fn roundTripperFunc) RoundTrip(r *http.Request) (*http.Response, error) {
+	return fn(r)
+}
+
+func mockRoundTripperFunc(resp interface{}) roundTripperFunc {
+	return func(r *http.Request) (*http.Response, error) {
+		raw, err := json.Marshal(&resp)
+		if err != nil {
+			return nil, err
+		}
+		resp := rawResponse{
```

Comment:
I think you are "hiding" `hiding` variable defined in line 621, and used in 623. It's confusing. Please rename one of the `resp` variables.